src/main/java/au/id/micolous/metrodroid/fragment/SupportedCardsFragment.kt

```diff
@@ -0,0 +1,182 @@
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.BaseExpandableListAdapter
+import android.widget.ImageView
+import android.widget.ListView
+import android.widget.TextView
+
+import au.id.micolous.farebot.R
+import au.id.micolous.metrodroid.MetrodroidApplication
+import au.id.micolous.metrodroid.activity.MainActivity
+import au.id.micolous.metrodroid.card.CardType
+import au.id.micolous.metrodroid.card.classic.ClassicAndroidReader
+import au.id.micolous.metrodroid.multi.Localizer
+import au.id.micolous.metrodroid.transit.CardInfo
+import au.id.micolous.metrodroid.transit.CardInfoRegistry
+import au.id.micolous.metrodroid.util.DrawableUtils
+import au.id.micolous.metrodroid.util.Preferences
+
+/**
+ * @author Eric Butler, Michael Farrell
+ */
+class SupportedCardsFragment : ExpandableListFragment() {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        listAdapter = CardsAdapter(context!!)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        if (item.itemId == android.R.id.home) {
+            val intent = Intent(context, MainActivity::class.java)
+            intent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP
+            startActivity(intent)
+            return true
+        }
+        return false
+    }
+
+    private class CardsAdapter internal constructor(val context: Context) : BaseExpandableListAdapter() {
+        val keyBundles: Set<String>? = ClassicAndroidReader.getKeyRetriever(context).forClassicStatic()?.allBundles
+        val cards = CardInfoRegistry.allCardsByRegion
+        private val mLayoutInflater: LayoutInflater = context.getSystemService(
+                    Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+
+        override fun getGroupCount(): Int = cards.size
+
+        override fun getGroup(i: Int) = cards[i].second
+
+        override fun isChildSelectable(i: Int, i1: Int): Boolean = false
+
+        override fun hasStableIds(): Boolean = true
+
+        override fun getChildrenCount(i: Int): Int = cards[i].second.size
+            
+        override fun getChild(parent: Int, child: Int): Any = cards[parent].second[child]
+
+        override fun getGroupId(i: Int): Long = (i.toLong() + 0x10)
+
+        override fun getChildId(parent: Int, child: Int): Long
+            = ((parent.toLong() shl 12) + child + 0x10000000)
+
+        override fun getGroupView(group: Int, isExpanded: Boolean, convertViewReuse: View?, parent: ViewGroup): View {
+            val convertView = convertViewReuse ?: mLayoutInflater.inflate(R.layout.supported_country_header,
+                        parent, false)
+            val textView1 = convertView!!.findViewById<TextView>(android.R.id.text1)
+            textView1.text = cards[group].first.translatedName
```

Comment:
Can we show the child count for each region?

The new list is a lot easier to navigate, but you no longer have a sense of scale about just how many cards there are in each region.